src/server/src/main/java/io/cassandrareaper/resources/RepairScheduleResource.java

```diff
@@ -520,4 +543,28 @@ private DateTime getNextActivationTime(Optional<String> scheduleTriggerTime) thr
     return nextActivation;
   }
 
+  /**
+   * Adjusts the DateTime object passed as argument to account for timezone differences. Reaper
+   * enforces UTC on all DateTime objects in the server, while the UI uses no specific timezone.
+   * This method allows to perform the appropriate conversion so that the user doesn't have to
+   * compute it.
+   *
+   * @param nextActivation The next activation time for the schedule, without timezone adjustments.
+   * @param clientTime Optional time as recorded on the client side.
+   * @return a DateTime object with adjusted time.
+   */
+  @VisibleForTesting
+  public static DateTime adjustTimeDifferenceOnNextActivation(
+      DateTime nextActivation, Optional<String> clientTime) {
+    DateTime adjustedNextActivation = nextActivation;
+
+    if (clientTime.isPresent()) {
+      DateTime clientLocalTime = DateTime.parse(clientTime.get());
+      Long timeDifference = DateTime.now().getMillis() - clientLocalTime.getMillis();
+      adjustedNextActivation = adjustedNextActivation.plusMillis(timeDifference.intValue());
```

Comment:
Isn't this making the presumption that the client is sending in the nextActiviation dateTime in its local timezone format?

The API requests that the `scheduleTriggerTime` argument be in UTC, ref https://github.com/thelastpickle/cassandra-reaper/blob/master/src/packaging/bin/spreaper#L236

Now if the client follows the API it breaks.